packages/neos-ui-sagas/src/CR/NodeOperations/reloadNodes.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * This file is part of the Neos.Neos.Ui package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+import {call, put, select} from 'redux-saga/effects';
+
+import {actions, selectors} from '@neos-project/neos-ui-redux-store';
+import {GlobalState} from '@neos-project/neos-ui-redux-store/src/System';
+import {DimensionCombination, Node, NodeContextPath, NodeMap, WorkspaceName} from '@neos-project/neos-ts-interfaces';
+import {AnyError} from '@neos-project/neos-ui-error';
+import backend from '@neos-project/neos-ui-backend-connector';
+// @ts-ignore
+import {getGuestFrameDocument} from '@neos-project/neos-ui-guest-frame/src/dom';
+
+// @TODO: This is a helper to gain type access to the available backend endpoints.
+// It shouldn't be necessary to do this, and this hack should be removed once a
+// better type API is available
+import {default as Endpoints, Routes} from '@neos-project/neos-ui-backend-connector/src/Endpoints';
+type Endpoints = ReturnType<typeof Endpoints>;
```

Comment:
hmm i had to read twice till i understood :)

i would favour to export the type just there and then add the comment there as well ... here it seems to be a little misplaced and will likely never be found :O

```js
import {EndpointsType, Routes} from '@neos-project/neos-ui-backend-connector/src/Endpoints';
```

Reply:
Will be solved as followup https://github.com/neos/neos-ui/pull/3759/files#r1575121070